vm/src/pyobject.rs

```diff
@@ -1368,6 +1361,15 @@ pub trait PyValue: fmt::Debug + PyThreadingConstraint + Sized + 'static {
     }
 }
 
+fn pyvalue_into_ref<T: PyValue>(value: T, cls: PyClassRef, vm: &VirtualMachine) -> PyRef<T> {
```

Comment:
can this function be a method of `PyValue`?

Reply:
Maybe, but I didn't put it because I just wanted to dedup some code; it wouldn't make sense to override it in an `impl PyValue`

Reply:
Though maybe it would, idk. I'll just do it.